packages/react/src/components/signature-pad/signature-pad-clear-trigger.tsx

```diff
@@ -0,0 +1,19 @@
+import { mergeProps } from '@zag-js/react'
+import { forwardRef } from 'react'
+
+import { type HTMLArkProps, ark } from '../factory'
+import { useSignaturePadContext } from './use-signature-pad-context'
+
+export interface SignaturePadClearTriggerProps extends HTMLArkProps<'button'> {}
+
+export const SignaturePadClearTrigger = forwardRef<
+  HTMLButtonElement,
+  SignaturePadClearTriggerProps
+>((props, ref) => {
+  const signature = useSignaturePadContext()
+  const mergedProps = mergeProps(signature.clearTriggerProps, props)
```

Comment:
In the `signature-pad.connect.ts` there are (for example) `getClearTriggerProps`. It would be best if you used them like below for proper type-check.

```suggestion
  const mergedProps = mergeProps(signature.getClearTriggerProps(), props)
```

Reply:
yeah I just noticed this changed on zag 0.54.0 😅